_data/projects/diode.json

```diff
@@ -1,113 +1,117 @@
-        "links": []
-    }, 
-    
-    {
-        "Title": "Boost Converter",
-        "year":"",
-        "image":"res/img/placeh.jpg",
-        "Abstract": "Solar panels essentially provide DC power and DC to DC converters are used to control voltage         levels and extract maximum power.We have designed a boost converter,         which steps up the input DC voltage from the panel.This can be used to charge a battery or can be connected to an         inverter to supply power back to the grid.PI control and Maximum Power Point Tracking were         simulated using Simulink, MATLAB.The hardware implementation was tested with a panel.         ",
-        "links": []
-    },
+        {
+            "Title": "Small scale radar",
+            "year":"2018 - 2019",
+            "image":"res/img/placeh.jpg",
+            "Abstract": "Radar is an object-detection system that uses radio waves to determine the range, angle, or velocity of objects.Radar was used as an acronym for RAdio Detection And Ranging.The aim of the project is to build a small scale version of such a RADAR system for educational and laboratory purposes.The radar system has coffee cans for transmit and receive antennae and components from Mini circuits.The system employs a ramp generator with a trigger output for transmission and present target is achieving range and doppler radar.Simulation of the radar system was done using Advanced Design System(ADS) software.For post processing, MATLAB is used to process the received data.The following experiments are planned and the MATLAB algorithms were implemented for the same - Doppler Vs TimeRange Vs TimeSAR imaging. ",
+            "links": []
+        }
+    ],
 
-    {
-        "Title": "Undersampling and reconstruction of MR images",
-        "year":"",
-        "image":"res/img/placeh.jpg",
-        "Abstract": "Magnetic resonance imaging, one of the foremost non-invasive imaging modalities, suffers from a slow data acquisition rate.Patients must remain in the scanner for extended periods of time, which causes them considerable discomfort, and may even delay a critical diagnosis. Reduced scan times would not only address these issues but also decrease the cost, which is a critical factor in the Indian scenario.This project compares the results obtained by applying compressed sensing and sparse dictionary learning algorithms for MR rimage reconstruction. ",
-        "links": []
-    },
+    "2018" : [
+        {
+            "Title": "Boost Converter",
```

Comment:
Are you sure the title for this should be Boost Converter? Looks like the first project in 2019 had the same title.

Reply:
Yeah, in the current website also it is like that only